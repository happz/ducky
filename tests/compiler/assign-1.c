void f()
{
  char c1, c2;

  c1 = c2;
}
