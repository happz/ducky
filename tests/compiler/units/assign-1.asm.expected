
  .include "defs.asm"
  .section .rodata


.L1_global:


  .text
  .global f

f:
  sub sp, 4
  push r11
  push r12


  lb r12, fp[-4]
  mov r11, r12

.L2_f_return:
  pop r12
  pop r11
  add sp, 4
  ret


_start:
  call &main
  int $INT_HALT
  .global _start
